mod-source-record-manager-server/src/main/java/org/folio/services/flowcontrol/RawRecordsFlowControlServiceImpl.java

```diff
@@ -0,0 +1,109 @@
+package org.folio.services.flowcontrol;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.folio.kafka.KafkaConsumerWrapper;
+import org.folio.verticle.consumers.consumerstorage.KafkaConsumersStorage;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import javax.annotation.PostConstruct;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+
+import static org.folio.rest.jaxrs.model.DataImportEventTypes.DI_RAW_RECORDS_CHUNK_READ;
+
+@Service
+public class RawRecordsFlowControlServiceImpl implements RawRecordsFlowControlService {
+  private static final Logger LOGGER = LogManager.getLogger();
+
+  @Value("${di.flow.max_simultaneous_records:100}")
+  private Integer maxSimultaneousRecords;
+  @Value("${di.flow.records_threshold:50}")
+  private Integer recordsThreshold;
+  @Value("${di.flow.control.enable:true}")
```

Comment:
nitpick: based on the scheme above i think this parameter should be `di.flow.enable`. I have not seen other parameters yet, so i may rescind this nitpick. This will hold especially if `enable` is the only option under `control`.

Reply:
`di.flow_control` might get your idea across.

Reply:
renamed all 3 params according params convention from mod-data-import: [Link](https://github.com/folio-org/mod-data-import/blob/1b0aa10019ed45a463e143e461104944b27e8687/src/main/java/org/folio/service/processing/reader/SourceReaderBuilder.java#L18)
Now they looks like this:
di.flow.control.max.simultaneous.records
di.flow.control.records.threshold
di.flow.control.enable

Reply:
looks good!